src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/SummaryJavadocCheck.java

```diff
@@ -237,11 +242,11 @@ private void validateUntaggedSummary(DetailNode ast) {
         }
         else if (!period.isEmpty()) {
             if (summaryDoc.contains(period)) {
-                final String firstSentence = getFirstSentenceOrNull(ast, period);
-                if (firstSentence == null) {
+                final Optional<String> firstSentence = getFirstSentence(ast, period);
+                if (firstSentence.isEmpty()) {
                     log(ast.getLineNumber(), MSG_SUMMARY_FIRST_SENTENCE);
                 }
-                else if (containsForbiddenFragment(firstSentence)) {
+                else if (containsForbiddenFragment(firstSentence.orElseThrow())) {
```

Comment:
why orElseThrow ?

Check should not throw exceptions, except for cases of wrong config and bad AST.

Reply:
fixed